tests/test_http.py

```diff
@@ -0,0 +1,22 @@
+"""Tests for the safir.http module.
+"""
+
+from aiohttp import web
+
+from safir.http import init_http_session
+
+
+async def test_init_http_session(aiohttp_client):
+    """Test basic use of init_http_session.
+    """
+
+    def create_app():
+        app = web.Application()
+        app.cleanup_ctx.append(init_http_session)
+        return app
+
+    app = create_app()
+    # Need to create the client itself to "start up" the app
+    client = await aiohttp_client(app)  # noqa: F841
```

Comment:
Nit: Why not drop the assignment so that you don't need the flake8 annotation? Or assert on the type of the `client` return value.